package.json

```
@@ -17,12 +17,18 @@
     "test:dev": "jest --watch",
     "test:e2e": "jest --config e2e/jest-e2e.json",
     "prerelease": "npm run build",
-    "release": "release-it"
+    "release": "release-it",
+    "---execution---": "",
+    "start": "nest start",
+    "start:dev": "nest start --watch",
+    "start:debug": "nest start --watch --debug"
   },
   "dependencies": {
     "@nestjs/mapped-types": "1.0.1",
+    "js-yaml": "^4.1.0",
```

let's pin this as dependencies upgrades are handled by Renovatebot

```suggestion
    "js-yaml": "4.1.0",
```